src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -789,6 +815,32 @@ protected theorem extractLsb_ofNat (x n : Nat) (hi lo : Nat) :
     getLsbD (extractLsb hi lo x) i = (i ≤ (hi-lo) && getLsbD x (lo+i)) := by
   simp [getLsbD, Nat.lt_succ]
 
+@[boolToPropSimps] theorem and_eq_decide (a b : Bool) : (a && b) = decide (a = true ∧ b = true) := by simp [Bool.decide_and]
+
+attribute [boolToPropSimps] decide_eq_true_iff
```

Comment:
Why is this in this file?

Reply:
Then we should not yet mark this for review, but try to get this resolved first.